library-best-practices/JLBP-1.md

```diff
@@ -2,13 +2,19 @@
 ------------------------------
 
 Use the minimum number of dependencies that is reasonable.
-Adding a dependency for a large amount of functionality may be ok,
-but avoid pulling in dependencies just to save a few lines of code,
-because every dependency of a library is a liability of both
-that library and that library's consumers.
+Every dependency of a library is a liability of both
+that library and its consumers.
+
+Adding a dependency for a large amount of functionality or
+for a complex functionality may be OK,
+but avoid pulling in dependencies just to save a few lines of code.
```

Comment:
perhaps "avoid adding a dependency just to save a few lines of code"

Reply:
done